build failed

Hello,
This project seems perfect for our usage (Robots powered by NUC with two realsense r200 camera and ROS).
But I can't manage to build this package, I get the following error:

`src/vpx-image-transport/src/streamer/webm-tools.patch:103: new blank line at EOF.
+
vpx-image-transport/src/streamer/CMakeFiles/webm-tools.dir/build.make:98: recipe for target 'vpx-image-transport/src/streamer/third_party/webm-tools/src/webm-tools-stamp/webm-tools-patch' failed
make[2]: *** [vpx-image-transport/src/streamer/third_party/webm-tools/src/webm-tools-stamp/webm-tools-patch] Error 1
`

--- a/src/streamer/CMakeLists.txt
+++ b/src/streamer/CMakeLists.txt
@@ -15,7 +15,7 @@ pkg_check_modules(PC_LIBVA REQUIRED libva)
 ExternalProject_Add(webm-tools
   PREFIX third_party/webm-tools
   GIT_REPOSITORY https://github.com/webmproject/webm-tools.git
-  GIT_TAG master
+  GIT_TAG 538dc2d55ea10d5507e1eb353b463bfc34f34b07
   PATCH_COMMAND git apply ${CMAKE_CURRENT_SOURCE_DIR}/webm-tools.patch
   BUILD_COMMAND make -j8
   INSTALL_COMMAND ""

